test/check-machines-disks

```diff
@@ -1501,6 +1501,7 @@ class TestMachinesDisks(VirtualMachinesCase):
         b.click("#delete-vm-subVmTest1-disks-vdc")
         b.wait_visible(".pf-c-modal-box")
         b.click(".pf-c-modal-box__footer button:contains(Remove)")
+        wait(lambda: "vdc" not in m.execute("virsh domblklist subVmTest1"), delay=1)
```

Comment:
How was that a race condition previously? It's already a `wait_*`. Could it be that it simply takes longer than 15s sometimes, and thus it needs a wait_timeout()?

That said, this extra check is actually nice -- it just has the wrong justification.

Reply:
Yeah, since this is a hot unplug, unplugging device from running VM might take longer than 15s since it needs the cooperation of the underlying VM OS.

And libvirt has a timeout on its hotunplug API of 5 seconds, and if it will not get a response from OS about hotunplug in 5seconds, it will just finish the API call, which results in the dialog being closed.

And thus `b.wait_not_present(".pf-c-modal-box")` alone is not enough